src/main/java/reposense/model/Format.java

```diff
@@ -9,7 +9,8 @@
  */
 public class Format {
     public static final List<String> DEFAULT_FORMAT_STRINGS = Arrays.asList(
-            "adoc", "cs", "css", "fxml", "gradle", "html", "java", "js", "json", "jsp", "md", "py", "tag", "xml");
+            "adoc", "cs", "css", "fxml", "gradle", "html", "java", "js", "json",
+            "jsp", "md", "py", "tag", "xml", "txt");
```

Comment:
hmm, interesting addition, seems like we missed out `txt` file. 
However, this PR does not warrant this fix; each PR should only target a single issue (unless they coincides, not in this sense though) to keep it history of changes clean.

Hence, please remove this change and open another issue for it.

Reply:
Opps 😱 

Try this:

```
config.setFormats(Format.convertStringsToFormats(Arrays.asList("txt")));
```